aspnetcore/host-and-deploy/aspnet-core-module.md

```diff
@@ -257,17 +263,35 @@ When using the out-of-process hosting model, the app might not shut down immedia
 ::: moniker-end
 
 ## Start-up error page
-
 ::: moniker range=">= aspnetcore-2.2"
 
-*Only applies to out-of-process hosting.*
+Both inprocess and outofprocess have custom error pages when they fail to start the application.
+
+If the ASP.NET Core module fails to find either the inprocess or out of process request handler, a *500.0 In-Process/Out-Of-Process Handler Load Failure* status code page appears.
+
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500-inproc.png)
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500-outofproc.png)
+
+For inprocess, if the ASP.NET Core module fails to start the application, a *500.30 Start Failure* status code page appears.
+
+![502.5 Process Failure Status Code Page](aspnet-core-module/_static/ANCMV2-500_30-inproc.png)
+
+For out of process, if the ASP.NET Core Module fails to launch the backend process or the backend process starts but fails to listen on the configured port, a *502.5 Process Failure* status code page appears.
```

Comment:
* "For out-of-process hosting if the ..."
* Dash in the error

Cut-'n-paste ...

> For out-of-process hosting if the ASP.NET Core Module fails to launch the backend process or the backend process starts but fails to listen on the configured port, a \*502.5 - Process Failure\* status code page appears.